abstractUserContextManager.py

```diff
@@ -0,0 +1,10 @@
+class abstractUserContextManager:
```

Comment:
Something that's worth discussing is what kind of style we're using for classes/variables/etc. I think for Python we capitalize class names normally, but I'm flexible to anything.